### DISCUSSION TITLE
View component not rendering in gatsby production build?

### DISCUSSION THREAD

**Discussion by frdclav** [adobe/react-spectrum#2535] [Q&A] [answered]
I am trying to play around with Gatsby and React Spectrum, but I am running into an odd issue: My 'gatsby develop' site is rendering differently from my production build. I am not too familiar with Spectrum yet, but it seems like during the production build, the View component is not rendering ?

Here is my repo: https://github.com/frdclav/golf-scorecards
prod site deployed on netlify: https://rods-golf-scorecards.netlify.app/

Steps to repro:
1. clone repo
2. in one terminal run `gatsby develop` -> this will run the site locally with the build where <View> is loading.
3. in another terminal run `gatsby build && gatsby serve` -> serves the production build of the site where <View> is not loading.

Please let me know what other info I can provide here, thanks
![Screen Shot 2021-11-09 at 11 39 46 PM](https://user-images.githubusercontent.com/54876035/141070903-ab72880c-84ae-4eea-bbc7-eba7839264d2.png)!

**Comment by reidbarber** [comment, marked as answer]
I think including the SSRProvider should fix this, as described [here](https://react-spectrum.adobe.com/react-spectrum/ssr.html):

```jsx
import {SSRProvider, Provider, defaultTheme} from '@adobe/react-spectrum';

<SSRProvider>
  <Provider theme={defaultTheme} locale={locale}>
    ...
  </Provider>
</SSRProvider>
```

I just tested a simple app locally with a View and the build seemed to work. If that doesn't fix the issue, feel free to leave a comment.

**Reply by frdclav** [reply]
Thank you!
